packages/fiori/src/themes/Page.css

```diff
@@ -2,6 +2,7 @@
 :host(:not([hidden])) {
     width: 100%;
     height: 100%;
+    display: inline-block;
```

Comment:
I would say `display: block`, or it specifically should be "inline-block"

Reply:
We are setting the display to ```inline-block``` on all of our host elements. If you think that there is something specific I can change it.

Reply:
Somehow thought that "block" is more suitable for a layout component like the Page. In theory, block elements start on a new line, inline-blocks - not.  For example the FCL is also "block".  That's why I commented it, but no specific issue at all.